addons/hr_timesheet/migrations/10.0.1.0/pre-migration.py

```diff
@@ -13,7 +13,7 @@
 
 def create_and_populate_department(cr):
     cr.execute('''
-       ALTER TABLE account_analytic_line ADD COLUMN department_id INT;
+       ALTER TABLE account_analytic_line ADD COLUMN IF NOT EXISTS department_id INT;
```

Comment:
Thanks! Can you add a docstring to this method mentioning the OCA module that provided the field for 9.0?